contribs/src/main/java/com/netflix/conductor/contribs/http/GenericHttpTask.java

```diff
@@ -228,7 +235,15 @@ private void setAuthorization(Input input) throws Exception {
 		AuthResponse response = auth.authorize();
 		input.getHeaders().put(HttpHeaders.AUTHORIZATION, "Bearer " + response.getAccessToken());
 	}
-
+	private void setAuthorizationContext(Input input,Map<String, Object> authorizationContext) throws Exception {
+		if(MapUtils.isNotEmpty(authorizationContext)) {
+			input.getHeaders().put("Authorization-Context", authorizationContext);
+		}
+		else
```

Comment:
For consistency use `} else {` here instead of using 3 separate lines.

Reply:
@heajuenhwang @hueys 
Else block styling updated